{"name": "test_vwo_login_negative", "status": "broken", "statusDetails": {"message": "RecursionError: maximum recursion depth exceeded", "trace": "setup = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"3b746203d1e5157e3dc5cb2dfb2ee33e\")>\n\n    @allure.epic(\"VWO login test\")\n    @allure.feature(\"TC#0 - VWO app negative test\")\n    @pytest.mark.negative\n    def test_vwo_login_negative(setup):\n        driver = setup\n        loginpage = LoginPage(driver)\n        loginpage.login_to_vwo(usr=\"admin@gmail.com\",pwd=\"admin@123\")\n        time.sleep(5)\n>       error_message = loginpage.get_error_msg_text()\n\nTests\\vwoLoginTests\\test_vwo_login.py:26: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\nTests\\PageObjects\\loginPage.py:58: in get_error_msg_text\n    return self.get_error_msg_text().text\nTests\\PageObjects\\loginPage.py:58: in get_error_msg_text\n    return self.get_error_msg_text().text\nE   RecursionError: maximum recursion depth exceeded\n!!! Recursion detected (same locals & position)"}, "start": 1740495978677, "stop": 1740495985097, "uuid": "98845d30-924a-4a97-be81-4b23d4424191", "historyId": "7e647765acc19a3a4c35ac49a86236c4", "testCaseId": "7e647765acc19a3a4c35ac49a86236c4", "fullName": "Tests.vwoLoginTests.test_vwo_login#test_vwo_login_negative", "labels": [{"name": "epic", "value": "VWO login test"}, {"name": "feature", "value": "TC#0 - VWO app negative test"}, {"name": "tag", "value": "negative"}, {"name": "parentSuite", "value": "Tests.vwoLoginTests"}, {"name": "suite", "value": "test_vwo_login"}, {"name": "host", "value": "DESKTOP-SJ9H8P2"}, {"name": "thread", "value": "11344-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Tests.vwoLoginTests.test_vwo_login"}]}